pkg/annotations/annotations.go

```diff
@@ -0,0 +1,66 @@
+package annotations
+
+const (
+	// Annotations carries the received Kubelet annotations
+	Annotations = "io.kubernetes.cri-o.Annotations"
```

Comment:
well, sorry to comment again :smile: I meant to have `io.cri-o.*` w/o the `kubernetes` in the middle, wdyt?